feat: Add custom distributed tracing docs for JS/Node (#8469)
getsentry/sentry-docs: src/platform-includes/distributed-tracing/custom-instrumentation/node.mdx

On this page, you will learn how to manually propagate trace information into and out of your Node.js application.

In the following cases, distributed tracing will be set up automatically for you:

- You have our <PlatformLink to="/performance/">performance monitoring feature</PlatformLink> turned on.

> - You are using one of the platforms that include distributed tracing out of the box:

Reviewer: ```suggestion
- You're using one of the platforms that include distributed tracing out of the box:
```

  - `@sentry/nextjs`
  - `@sentry/sveltekit`
  - `@sentry/remix`
  - `@sentry/astro`

> If you are using a different package, and have not enabled performance monitoring, you can manually set up your application for distributed tracing to work.

Reviewer: ```suggestion
If you're using a different package, and haven't enabled performance monitoring, you can manually set up distributed tracing in your application.
```

## Enabling Distributed Tracing without Performance

You can setup the `Http` integration in combination with `tracesSampleRate: 0` to enable distributed tracing without performance monitoring:

```javascript
Sentry.init({
  dsn: "___PUBLIC_DSN___",
  integrations: [new Sentry.Integrations.Http({ tracing: true })],
  tracesSampleRate: 0,
});
```

## Enabling Distributed Tracing without `Http` Integration

If you do not want to use the `Http` integration in conjuction with `tracing: true`, you can manually extract and inject tracing information into your application. For this, you must:

- Extract and store incoming tracing information from incoming request headers or similar.
- Inject tracing information to any outgoing requests.

> To learn more, see our <PlatformLink to="/usage/distributed-tracing/">Distributed Tracing</PlatformLink> docs.

Reviewer: ```suggestion
To learn more about distributed tracing, see our <PlatformLink to="/usage/distributed-tracing/">Distributed Tracing</PlatformLink> docs.
```

### Extracting Incoming Tracing Information

> Incoming tracing information has to be extracted and stored in memory for later use. Sentry provides the `continueTrace()` function to help you with this. Incoming tracing information can come from different places:

Reviewer: ```suggestion
You must extract and store incoming tracing information in memory for later use. Sentry provides the `continueTrace()` function to help you with this. Incoming tracing information can come from different places:
```

On this page you will learn how to manually propagate trace information into and out of your JavaScript application.

In the following cases, distributed tracing will be set up automatically for you:

- You have our <PlatformLink to="/performance/">performance monitoring feature</PlatformLink> turned on.
- You are using one of the platforms that include distributed tracing out of the box:
  - `@sentry/nextjs`
  - `@sentry/sveltekit`
  - `@sentry/remix`
  - `@sentry/astro`

If you are using a different package, and have not enabled performance monitoring, you can manually set up your application for distributed tracing to work.

## Enabling Distributed Tracing without Performance

You can setup the `Http` integration in combination with `tracesSampleRate: 0` to enable distributed tracing without performance monitoring:

```javascript
Sentry.init({
  dsn: "___PUBLIC_DSN___",
  integrations: [new Sentry.Integrations.Http({ tracing: true })],
  tracesSampleRate: 0,
});

> ```

Author: good call, but we should prob. also mention that undici only works on node 18+ then, right? 🤔

> ```

Author: Updated this! 👍
